docs/versioned_docs/version-2.11.0/deprecated/releases-1x.md

```diff
@@ -0,0 +1,200 @@
+---
+id: releases-1x
+title: OpenEBS 1.x Deprecated Releases
+---
+
+## 1.7.0 - Feb 15 2020
+
+**Change summary:**
+- Fixes an issue where Jiva Replicas could get stuck in WO or NA state, when the size of the replica data grows beyond 300GB.
+- Fixes an issue where unused custom resources from older versions are left in the etcd, even after openebs is upgraded. 
+- Fixes an issue where cleanup of Jiva volumes on OpenShift 4.2 environment was failing.
+- Fixes an issue where custom resources used by cStor Volumes fail to get deleted when the underlying pool was removed prior to deleting the volumes.
+- Fixes an issue where a cStor Volume Replica would be incorrectly marked as invalid due to a race condition caused between a terminating and its corresponding newly launched pool pods. 
+
+**Alpha Features**
+
+- Support for generating automated ARM builds for NDM.
+- Support for managing snapshot and clones of ZFS Local PV.
+- Support for setting up PDB and PriorityClasses on cStor Pool Pods. Increasing the e2e coverage and fixing the issue uncovered.
+- Support for resizing Jiva Volume via CSI driver and other bug fixes.
+
+**Additional details:**
+
+- [Release Notes](https://github.com/openebs/openebs/releases/tag/1.7.0)
+- [Upgrade Steps](/v170/docs/next/upgrade.html)
+
+## 1.6.0 - Jan 15 2020
+
+**Change summary:**
+- Add support for building cStor on ARM builds by moving cstor-pool-mgmt, cstor-volume-mgmt, maya-exporter and cspi-mgmt build scripts to their specific folders and add arm build scripts.
+- Add support for provisioning Local PV hostpath when the nodes are tainted. It also handles the implementation of Local PV helper pods where the cleanup pod will be launched with the tolerations.
+- Enhance the logging mechanism for cStor pool pods during pool import time. These changes of logs will help to identify the existence of bad disk on the node.
+- Support for enabling core dump by adding an ENV variable ENABLE_COREDUMP= “1” for cStor pool pod to control whether cores need to be dumped in case of process crashes. By default, dumping cores will be disabled. Make sure this environment variable is not enabled if mountPoint of `SparseDir` has been changed in CAS Templates.
+- Enhance upgrade logs by providing pool and volume status information during the upgrade and also helps in estimating the time taken for deployment pods to come up.
+-  Improves Jiva rebuilding process by check pointing the io numbers. Now only those snapshots will be synced which has less no of io’s.
+- Fixes an issue with Jiva controller by removing WO replica if new replica with greater revision count get added to controller.
+- Disable core dump in NDM daemon by default. This can be enabled by setting an ENV variable `ENABLE_COREDUMP` to `1`. Core files will be stored inside /var/openebs/ndm/core.
```

Comment:
```suggestion
- Disable core dump in NDM daemon by default. This can be enabled by setting an ENV variable `ENABLE_COREDUMP` to `1`. Core files will be stored inside `/var/openebs/ndm/core`.
```